databricks_cli/pipelines/cli.py

```diff
@@ -186,20 +188,25 @@ def list_cli(api_client):
 @provide_api_client
 def reset_cli(api_client, pipeline_id):
     """
-    Resets a delta pipeline by truncating tables and creating new checkpoint folders so data is
-    reprocessed from scratch.
+    [Deprecated] Use the "start --full-refresh" command instead.
+
+    Resets a pipeline by truncating tables and creating new checkpoint folders so that data is
+    reprocessed from the beginning.
 
     Usage:
 
     databricks pipelines reset --pipeline-id 1234
     """
+    click.echo("DeprecationWarning: the \"reset\" command is deprecated, " +
+               "use the \"start --full-refresh\" command instead.")
     _validate_pipeline_id(pipeline_id)
-    PipelinesApi(api_client).reset(pipeline_id)
-    click.echo("Reset triggered for pipeline {}".format(pipeline_id))
+    PipelinesApi(api_client).start_update(pipeline_id, full_refresh=True)
+    click.echo("Reset triggered for pipeline {}.".format(pipeline_id))
```

Comment:
I don't think "triggered" is a good word here. Since we recommend to use `start --full-refresh` here, should the message be the same? That is, `Started an update for pipeline {id}.`

Reply:
changed to a similar message across start/reset/run commands with update_id and mentioning if it's a full_refresh.